src/__tests__/sagas/forecasts.test.js

```diff
@@ -0,0 +1,26 @@
+import { put, call } from 'redux-saga/effects';
+import { fetchForecastByGeolocation } from '../../sagas/forecasts';
+import { testSaga } from 'redux-saga-test-plan';
+
+import * as types from '../../constants';
+import * as utils from '../../utils';
+import * as api from '../../api';
+
+describe('fetchForecastByGeolocation', () => {
+  it('requests the forecast data and dispatches actions', () => {
+    const saga = testSaga(fetchForecastByGeolocation);
+
+    saga
+      .next()
+      .put({ type: types.FETCH_FORECAST_START })
+      .next()
+      .call(utils.getUserLocation)
+      .next('London')
+      .call(api.get5DaysForecastByCoords, 'London')
```

Comment:
No expectation?

Reply:
This is not finished yet and the syntax with no expectations comes from the redux-saga-test-plan package. I will go back to the expect syntax but I was trying some things out and thought that this may help with assertions. 

I need a way to mock external API calls (working on it).